tests/utils.py

```diff
@@ -30,19 +30,24 @@ def setup_task(
             "state_reason": "state_reason",
             "state_history": [],
             "from_index": index,
+            "bundles": map_or_op,
```

Comment:
why are these changes meddled with this commit?

Reply:
Tests needed to be fixed due to iiblib changes